packages/devtools_app/lib/src/shared/device_dialog.dart

```diff
@@ -38,9 +38,7 @@ class DeviceDialog extends StatelessWidget {
 
     final vm = serviceManager.vm;
 
-    if (vm == null) return const SizedBox();
```

Comment:
why are we no longer checking for a null vm?

Reply:
It should be. It is set to null in vmServiceClosed: https://github.com/flutter/devtools/blob/master/packages/devtools_app/lib/src/shared/service_manager.dart#L332

Reply:
right. ServiceManager is not null safe yet. Thanks for the catch.